src/System.Web.HttpUtility/tests/HttpUtility/HttpUtilityTest.cs

```diff
@@ -412,6 +412,7 @@ public void UrlDecode(string decoded, string encoded)
 
         [Theory]
         [MemberData(nameof(UrlDecodeData))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Full framework assumes the current codepage is UTF8")]
```

Comment:
I'm not sure which one is the case here but definitely there is some difference in Desktop since I tried this in a Console app and it didn't decode the inputs as it should (%u0061 -> a) it just leaves %u0061 as is.

I will go ahead and split it into 2 tests.

@ianhays do you know more about this?